apis/dispatch.php

```diff
@@ -153,10 +154,8 @@
     exit();
 }
 
-if ($isLocalApi) {
-    // Ensure that a local process does not hold up other processes
-    session_write_close();
-}
+// TODO - Goal in future is to be able to remove this along with the above $sessionAllowWrite flag in the future
+session_write_close();
```

Comment:
Can't do this when not $isLocalApi since need to be able to set session variables when token is success (to set pid and puuid for portal api and authUser and authUserID for non-portal api). This is not a big deal since api only works on one session_start(), so won't gain anything from doing this for api (unless is $isLocalApi, then need to do it and it will be ok in that case).

TODO:
Will revert changes in above code block

Reply:
@bradymiller Did u say [remind urself with the github reminders app](https://probot.github.io/apps/reminders/)? xD